db-service/lib/cqn2sql.js

```diff
@@ -494,21 +489,6 @@ class CQN2SQLRenderer {
       !!q.UPSERT,
     )
     const extraction = extractions
-      .map(c => {
-        const element = elements?.[c.name]
-        if (element?.['@cds.extension']) {
-          return false
-        }
-        if (c.name === 'extensions__') {
-          const merges = extractions.filter(c => elements?.[c.name]?.['@cds.extension'])
-          if (merges.length) {
-            c.sql = `json_set(ifnull(${c.sql},'{}'),${merges.map(
-              c => this.string('$."' + c.name + '"') + ',' + c.sql,
-            )})`
-          }
-        }
-        return c
-      })
       .filter(a => a)
```

Comment:
@BobdenOs can we also remove that filter?

Reply:
The reason for the `filter` is line 500 which is being removed. This change should actually just rename the place where `extraction` is used to `extractions`.